wheel/metadata.py

```diff
@@ -32,7 +32,9 @@
 
 # Wheel itself is probably the only program that uses non-extras markers
```

Comment:
This is no longer true. Markers are widely used but wheel still needs the extra market at the end only (used in a converter)